### DISCUSSION TITLE
Tampilan angka infobox dashboard

### DISCUSSION THREAD

**Discussion by Samgoesniady** [teamalgoritma/community#444] [Q&A] [answered]
Selamat siang kak maaf ijin bertanya bagaimana cara agar membuat tampilan numeric di infobox lebih mudah dibaca jika angkanya mencapai ratusan juta? Adakah format untuk membuat angkanya lebih mudah dibaca? Terima kasih
![IMG-20221106-WA0003](https://user-images.githubusercontent.com/100010379/200150566-ac58230e-eb35-4515-b5d8-75f651021cec.jpg)

**Comment by divakartika** [comment, marked as answer]
Dear @Samgoesniady ,
Untuk membuat tampilan numerik menjadi lebih mudah terbaca, dapat digunakan fungsi `comma()` dari library `scales`. Untuk penggunaannya dapat langsung dimasukkan nilai numerik ke dalam fungsi `comma()` seperti `comma(926548000)`. 

Dokumentasi selengkapnya mengenai fungsi `comma()`: https://scales.r-lib.org/reference/number_format.html

Semoga informasi ini dapat membantu. Bila ternyata informasi ini dapat menjawab pertanyaan Anda, Anda dapat “_Mark as answer_” jawaban ini.

**Reply by Samgoesniady** [reply]
Sudah terjawab kak terima kasih
